### PR TITLE
SwapnilMane141.json

### PR DESCRIPTION
**Pull request by SwapnilMane141**

### CODE REVIEW

File: data/users/SwapnilMane141.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Swapnil Mane",
+  "bio": "Frontend Web Developer & Data Extraction Expert",
+  "image": "https://avatars.githubusercontent.com/u/66281302?v=4",
+  "categories": [
+    {
+      "slug": "programming | web-development | web scraping",
```
Comment (RakeshSangem) [inline data/users/SwapnilMane141.json:7]: Hey @SwapnilMane141 , please use any one of the slug. You can refer slug's from `public/categories.json`

File: data/users/SwapnilMane141.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Swapnil Mane",
+  "bio": "Frontend Web Developer & Data Extraction Expert",
```
Comment (RakeshSangem) [inline data/users/SwapnilMane141.json:3]: Would be better if you can pick one. Short description preferred like `web developer` or `Frontend Developer`

File: data/users/SwapnilMane141.json (hunk 1 above)
Comment (SwapnilMane141) [reply data/users/SwapnilMane141.json:7]: Sure... Will make the changes!

**Review by RakeshSangem — APPROVED** [review]
Thanks for the changes @SwapnilMane141 😊

**Review by codewithvoid — APPROVED** [review]

**Comment by codewithvoid** [comment]
Thanks for volunteering @SwapnilMane141 :)

**Pull request merged**